tests/bench.py

```
@@ -0,0 +1,60 @@
+
+def setup_module(module):
+    global redis_server, client, g, shutdown
+    target = os.environ.get("TARGET",
+                            "target/release/libfalkordb.dylib" if platform.system() == "Darwin" else "target/release/libfalkordb.so")
+    r = Redis()
+    try:
+        r.ping()
+        client = FalkorDB()
+        g = client.select_graph("test")
+        return
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)